<launch>
	<node pkg="joy" name="joy_node" type="joy_node">
    <param name="dev"              value="/dev/input/js1"   type="string"/>
    <param name="autorepeat_rate"  value="12.0"/>
  </node>
	
	<node pkg = "ohm_bob_remote" type = "joy_node" name = "remote_control" output="screen">
		<param name = "joy_topic" type = "string" value = "joy" />
		<param name = "vel_topic" type = "string" value = "vel/cmd" />
		<param name = "controller_type" type = "string" value = "ps3_sixad" />
		<param name = "lin_vel_thresh" type = "double" value = "0.5" />
		<param name = "ang_vel_thresh" type = "double" value = "0.5" />
	</node>
	
	<node pkg = "ohm_bob_remote" type = "velocity_switch" name = "velocity_switch" output = "screen"/>
	
</launch>

